source/docs/steeltoe-introduction.html.markdown

```diff
@@ -61,122 +58,139 @@ Below is an example NuGet.config file you can edit and use when developing appli
+>CAUTION: This is a BREAKING change, as it now requires that you MUST explicitly include the packages and versions of the RabbitMQ code you wish to use in your application.
 
-Also with this release, the connector has been updated to properly work with SSL/TLS based connections.  Prior to this release, the connector did not support using the 'amqps' URL scheme.
+Also, with this release, the connector has been updated to properly work with SSL/TLS based connections. Prior to this release, the connector did not support using the 'amqps' URL scheme.
 
 ### Redis Connector
 
-Starting with version 2.0.0 of Steeltoe, the direct dependencies on Redis packages have been removed. The Steeltoe connector works with the StackExchange libraries or `Microsoft.Extensions.Caching.Redis`. You include the package and version you prefer and use the appropriate methods to wire it up. See the Redis Connector documentation for more information.
+Starting with version 2.0.0 of Steeltoe, the direct dependencies on Redis packages have been removed. The Steeltoe connector works with the StackExchange libraries or `Microsoft.Extensions.Caching.Redis`. You include the package and version you prefer and use the appropriate methods to connect it. See the Redis Connector documentation for more information.
 
 ## 2.5 Discovery
 
-In version 1.x of Steeltoe, there were 3 packages, in version 2.0.0 there are 9: `Steeltoe.Discovery.ClientAutofac`, `Steeltoe.Discovery.ClientCore`, `Steeltoe.Discovery.Eureka.ClientBase`, `Pivotal.Discovery.ClientCore`, `Pivotal.Discovery.ClientAutofac`, `Pivotal.Discovery.Eureka.ClientBase`. Be sure to use the `Pivotal.&ast;` packages if you are deploying your application to Pivotal Cloud Foundry. Version 2.0.0 also includes support for dynamic update of configuration options.
+In version 1.x of Steeltoe, there were three packages. In version 2.0.0 there are six packages:
+
+* `Steeltoe.Discovery.ClientAutofac`
+* `Steeltoe.Discovery.ClientCore`
+* `Steeltoe.Discovery.Eureka.ClientBase`
+* `Pivotal.Discovery.ClientCore`
+* `Pivotal.Discovery.ClientAutofac`
+* `Pivotal.Discovery.Eureka.ClientBase`.
+
+Be sure to use the `Pivotal.&ast;` packages if you deploy your application to Pivotal Cloud Foundry. Version 2.0.0 also includes support for dynamic update of configuration options.
 
 ### Eureka Version
 
-Steeltoe has implemented a [Eureka 1.0 client](https://github.com/Netflix/eureka/wiki), not a 2.0 client. Eureka 2.0 is still a work in progress and is expected to have significant updates to its architecture and public API. At some point in time, we will examine a 2.0 implementation.
+Steeltoe has implemented a [Eureka 1.0 client](https://github.com/Netflix/eureka/wiki), not a 2.0 client. Eureka 2.0 is still a work in progress and is expected to have significant updates to its architecture and public API. At some point in time, we may examine a 2.0 implementation.
```

Comment:
I thought we wanted to remove any suggestion of future plans (see last sentence)